lib/ZnapZend/ZFS.pm

```diff
@@ -122,6 +123,30 @@ sub snapshotExists {
     return grep { $snapshot eq $_ } @snapshots;
 }
 
+sub createDataSet {
+    my $self = shift;
+    my $dataSet = shift;
+    my $remote;
+
+    #just in case if someone aks to check '';
+    return 0 if !$dataSet;
+
```

Comment:
if znapzend asks znapzend todo something stupid it should dy a horrible dath, crying bloody morder while doing so